.github/workflows/docker.yml

```diff
@@ -2,37 +2,15 @@ name: Build & Push
 #  Build & Push rebuilds the tendermint docker image on every push to master and creation of tags
 # and pushes the image to https://hub.docker.com/r/interchainio/simapp/tags
 on:
-  pull_request:
-  push:
-    branches:
-      - master
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
-      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+  release:
+    types:
+      - published
 
 jobs:
   build:
-    runs-on: ubuntu-latest
```

Comment:
Why are we not using latest?

Reply:
So we have a stable build environment that doesn't change when GitHub feels like it, but when we feel like it. `ubuntu-latest` is current 20.04 in GitHub Actions anyway.